[ 2025-03-02 01:23:01,173 ] 32 root - ERROR - \u274c Error loading object from artifacts/best_model.pkl: No module named '_loss'
[ 2025-03-02 01:23:01,176 ] 36 root - ERROR - \u274c Failed to load model: Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\utils.py] line number [29] error message[No module named '_loss']
[ 2025-03-02 01:23:01,177 ] 74 root - ERROR - \u274c Error during prediction: Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\pipelines\prediction_pipeline.py] line number [32] error message[Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\utils.py] line number [29] error message[No module named '_loss']]
[ 2025-03-02 01:23:01,178 ] 61 root - ERROR - \u274c Prediction failed
